### ISSUE TITLE
Inability to select media compression before sending (resulting in wasting time & bw maxing out the media repo)

### ISSUE THREAD

**Issue by ara4n** [element-hq/element-x-ios#2783]
### Steps to reproduce

1. Try to send someone a 1GB video recorded on phone
2. EX struggles for ages and then craps out

#### What did you expect?

 * Prompt me to select whether to compress the video before sending it, so I can decide how much bandwidth to chew (and how best to fit in the media repo’s limits)
 * Never try to send media bigger than the max of the media repo

#### What happened instead?

A miseraly crap experience

### Application version

592

### Will you send logs?

No

**Comment by Kimiblock** [comment]
This affects many small homeservers where bandwidth is very limited.

**Comment by giomfo** [comment]
@stefanceriu can we consider this done since you added the Optimise media quality option in Advanced settings?

<img width="820" alt="Image" src="https://github.com/user-attachments/assets/f8d953ef-19c8-409f-ba4e-784ac9581f86" />

**Comment by stefanceriu** [comment]
Yes, I believe that's fair. Let's close this one and handle (eventual) UX changes in https://github.com/element-hq/element-x-ios/issues/3812
